docs/usage.md

```diff
@@ -14,39 +12,23 @@ You will need to create a samplesheet with information about the samples you wou
-sample,fastq_1,fastq_2
-CONTROL_REP1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz
-CONTROL_REP2,AEG588A2_S2_L002_R1_001.fastq.gz,AEG588A2_S2_L002_R2_001.fastq.gz
-CONTROL_REP3,AEG588A3_S3_L002_R1_001.fastq.gz,AEG588A3_S3_L002_R2_001.fastq.gz
-TREATMENT_REP1,AEG588A4_S4_L003_R1_001.fastq.gz,
-TREATMENT_REP2,AEG588A5_S5_L003_R1_001.fastq.gz,
-TREATMENT_REP3,AEG588A6_S6_L003_R1_001.fastq.gz,
-TREATMENT_REP3,AEG588A6_S6_L004_R1_001.fastq.gz,
+patient,sample,variantcaller,vcf,vcf_tbi,varianttype
+A,t5,strelka,https://github.com/qbic-pipelines/test-datasets/raw/refs/heads/main/variantconsensus/tumor_5_vs_normal_5.strelka.somatic_indels_VEP.ann.vcf.gz,https://github.com/qbic-pipelines/test-datasets/raw/refs/heads/main/variantconsensus/tumor_5_vs_normal_5.strelka.somatic_indels_VEP.ann.vcf.gz.tbi,indels
 ```
 
-| Column    | Description                                                                                                                                                                            |
-| --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `sample`  | Custom sample name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`). |
-| `fastq_1` | Full path to FastQ file for Illumina short reads 1. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
-| `fastq_2` | Full path to FastQ file for Illumina short reads 2. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
+A description of the columns can be found in the table below:
+
+| Column          | Description                                                                                                       |
+| --------------- | ----------------------------------------------------------------------------------------------------------------- |
+| `patient`       | Unique identifier for the patient.                                                                                |
+| `sample`        | Custom sample name. This entry will be identical for variant calling runs from the same sample.                   |
+| `variantcaller` | The name of the variant caller used to generate the VCF file.                                                     |
+| `vcf`           | Full URL or path to the VCF file containing variant calls. File must be gzipped and have the extension `.vcf.gz`. |
+| `vcf_tbi`       | Full URL or path to the index file for the VCF file. File must have the extension `.vcf.gz.tbi`.                  |
+| `varianttype`   | Type of variants contained in the VCF file (e.g., `indels`, `snvs` or `both`.).                                   |
```

Comment:
```suggestion
| `varianttype`   | [Optional] type of variants contained in the VCF file (e.g., `indels`, `snvs` or `both`.).                                   |
```

Reply:
Can you clarify why this info is not in the `nextflow.schema` but just in `assets/schema_input.json`?

Reply:
I am just wondering why the info about the required columns is just mentioned in the `assets/schema_input.json`, but not in the `nextflow_schema.json`. In the  `schema_input.json` it seems as if `varianttype` is only optional and not required.